python/glow/gwas/approx_firth.py

```diff
@@ -25,22 +24,31 @@ class FirthFit:
 class FirthStatistics:
     effect: Float
     stderror: Float
-    tvalue: Float
+    chisq: Float
```

Comment:
What is this renaming for?

Reply:
Same reason as the other tvalue -> chisq renamings